src/v/cluster/health_monitor_types.h

```diff
@@ -71,6 +73,16 @@ struct topic_status {
     friend std::ostream& operator<<(std::ostream&, const topic_status&);
     friend bool operator==(const topic_status&, const topic_status&) = default;
 };
+
+struct node_local_state {
```

Comment:
This could use a comment explaining what "node local" means in this context.  I think I grok it (node local is stuff that isn't part of any distributed logic, unlike the topic status or node_id), but the naming's not completely intuitive (I do not have a better name suggestion!)

Reply:
Good call.. I'll add a hint to look at the bigger description (next to the Figlet ASCII which I'll remove).